doc/schema/schema.md

```diff
@@ -68,16 +68,18 @@ Samples from the posterior distribution p(theta|y).
 ### `sample_stats`
 Information and diagnostics for each `posterior` sample, provided by the inference backend. It may vary depending on the algorithm used by the backend (i.e. an affine invariant sampler has no energy associated). The name convention used for `sample_stats` variables is the following:
 * `lp`: (unnormalized) log probability for sample
-* `step_size`
-* `step_size_bar`
+* `step_size`: the step size used for this sample
```

Comment:
Can you clarify whether this is the jittered or unjittered step size (where jittering is done)?

Reply:
e.g. for Turing this is _jittered_ step size.